app/src/main/java/com/android/sooz/filterapp/SpeciesDetailActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.android.sooz.filterapp;
+
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class SpeciesDetailActivity extends StarWarsSpeciesListActivity {
```

Comment:
This might be better extended from ThemeHeaderFragmentActivity, as it isn't really very related to the List.